src/Benchmark/nlog.config

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<nlog xmlns="http://www.nlog-project.org/schemas/NLog.xsd"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  autoReload="true">
+  
+  <extensions>
+    <add assembly="NLog.StructuredLogging.Json" />
+  </extensions>
+  
+  <variable name="log_dir" value="${basedir}..\logs" />
+  <variable name="archive_dir" value="${log_dir}\archive" />
+  <variable name="human_readable_layout" value="${longdate}|${level}|${logger}|${message}|${onexception:EXCEPTION OCCURRED\:${exception:format=type,message,method,stackTrace:maxInnerExceptionLevel=5:innerFormat=shortType,message,method,stackTrace}}" />
+
+  <targets async="true">
```

Comment:
When doing benchmark to discover performance issues, then it is better to remove `async="true"`